### PR TITLE
Create simple redundant averaging script

### PR DESCRIPTION
**Pull request by jsdillon**
This PR creates a very simple script for redundantly averaging data that's already been calibrated. Possible future improvements include better memory management via partial i/o and/or moving parts of this script into its own unit-tested module.

### CODE REVIEW

File: scripts/red_average.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2019 the HERA Project
```
Comment (aewallwi) [inline scripts/red_average.py:3]: Maybe update the date.

File: scripts/red_average.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# Licensed under the MIT License
+
+"""Command-line script for redundant averaging of (calibrated) data. To calibrate and then average, try apply_cal.py"""
+
+import sys
+import argparse
+from hera_cal import io, utils, redcal
+
+# Parse arguments
+a = argparse.ArgumentParser(description="Redundantly average a data file that's already been calibrated.")
```
Comment (aewallwi) [inline scripts/red_average.py:13]: maybe consider using a non-single character filename like `a`.

**Review by aewallwi — CHANGES_REQUESTED** [review]
Looks good to me. Minor style changes to consider.

**Review by aewallwi — APPROVED** [review]

File: scripts/red_average.py (hunk 1 above)
Comment (jsdillon) [reply scripts/red_average.py]: Done.

File: scripts/red_average.py (hunk 2 above)
Comment (jsdillon) [reply scripts/red_average.py]: Done.

**Pull request merged**